src/connection.js

```diff
@@ -986,6 +1023,7 @@ class Connection extends EventEmitter {
       }
 
       this.request = request;
+      this.request.connection = this;
```

Comment:
> The difference, at least as I see it, is the coupling.

Yes, that's true. But I would like to finish my work on this feature now and work on other things. Can't we agree that the current implementation is good enough? There is always something that can be improved and de-coupling like this has not been a design principle in Tedious until now.